### ISSUE TITLE
New profession: Personal Assistant

### ISSUE THREAD

**Issue by mklosowski** [ldtteam/minecolonies-features#182]
### Prerequisites 

- [x] I am running the latest **alpha** version of Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my feature does not already exist.
- [x] I made sure that **this feature request is not a duplicate of any existing feature request.**

### Context 

- Minecraft Version: 1.16.2
- MineColonies Version: minecolonies-0.13.366-ALPHA-universal.jar

## Is your feature request related to a problem?
No.

## Feature description
Profession: Personal Assistant
Hut block: Desk - requires Book & Quill and Water Bottle to create hut block
A personal assistant acts as a go-between for the player and the colony to request and keep in stock items that the player might need on a regular basis, e.g. food, armor, tools, potions (with the Apothecary).

### Notes or related things
Could be unlocked as part of the Civilian research tree, or maybe when the colony reaches 10 citizens.
The Personal Assistant block can be placed anywhere, e.g. at the player's base (assuming that is within the boundaries of the colony) or in or near the colony's warehouse.
The hut block GUI would allow the player to request items, e.g. diamond sword or cooked chicken, and set a minimum number the Personal Assistant would try to keep in stock.

---
## Viewers

* Add a thumbs-up to the feature request to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Mekle001** [comment]
This is basically adding prestock to the postbox, yes?

**Comment by mklosowski** [comment]
Yes, that sums up the major functionality. I was envisioning a colonist helper who might live in your base and putter around appearing to keep things in order. Kind of a npc postbox+.

**Comment by mklosowski** [comment]
Also, not sure whether this is possible, but if you request a set of diamond armor, perhaps the assistant could put it on an armor stand as the pieces are delivered. They you could take them as needed.
